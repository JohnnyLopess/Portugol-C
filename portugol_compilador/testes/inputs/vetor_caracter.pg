programa {
    funcao inicio() {
        caracter v[3]
        v[0] = 65
        v[1] = 66
        escreva(v[0])
        escreva(v[1])
    }
}
