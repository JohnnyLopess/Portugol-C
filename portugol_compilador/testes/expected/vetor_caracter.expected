#include <stdio.h>

int main() {
    char v[3];
    v[0] = 65;
    v[1] = 66;
    printf("%c\n", v[0]);
    printf("%c\n", v[1]);
    return 0;
}
